hack/chains/kaniko-cosign-verify.sh

```diff
@@ -29,19 +31,67 @@ title "Image digest from task result"
 kubectl get $TASKRUN_NAME -o jsonpath="{.status.taskResults[?(@.name == \"IMAGE_DIGEST\")].value}"
 echo
 
-title "Cosign verify the image"
-# Save the output data to a file so we can look at it later
-# (Actually we could just pipe it to jq because the text goes to stderr I think..?)
-show-then-run "cosign verify --key $SIG_KEY $IMAGE_URL --output-file /tmp/verify.out"
+if $INTERNAL_REGISTRY; then
+  title "Cosign verify the image - using Tekton task"
+  COSIGN_VERIFY_TASK_NAME="cosign-verify-$(openssl rand --hex 10)"
```

Comment:
Nice, I was looking for a simple random string creator in create-signing-secret.